src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveDefendantResponseTest.java

```diff
@@ -60,6 +61,22 @@ public void shouldReturnNewlyCreatedDefendantResponse() throws Exception {
             .contains(Optional.of(response));
     }
 
+    @Test
```

Comment:
IMO to keep language consistent we should say `shouldReturnClaimWithNewlyCreatedPartAdmission`